src/simplewallet/simplewallet.cpp

```diff
@@ -9076,7 +9105,7 @@ void simple_wallet::commit_or_save(std::vector<tools::wallet2::pending_tx>& ptx_
       tx_to_blob(ptx.tx, blob);
       const std::string blob_hex = epee::string_tools::buff_to_hex_nodelimer(blob);
       const std::string filename = "raw_monero_tx" + (ptx_vector.size() == 1 ? "" : ("_" + std::to_string(i++)));
-      if (epee::file_io_utils::save_string_to_file(filename, blob_hex))
+      if (m_wallet->save_to_file(filename, blob_hex))
```

Comment:
That already seems to be a hex dump of the data.

Reply:
Good catch, thanks. Fixed.

Reply:
Turns out there's several such cases - get_tx_proof, get_spend_proof, and get_reserve_proof all return base58 encoded strings.

Now we have a bunch of occasions where we call `m_wallet->save_to_file` and a bunch of occasions where we call `epee::file_io_utils::save_string_to_file`. Maybe adding a new argument for `save_to_file` like `is_hex` or `is_ascii` would be more appropriate?

Reply:
Fixed.